Radzen.Blazor/RadzenYearView.razor.cs

```diff
@@ -0,0 +1,119 @@
+using Microsoft.AspNetCore.Components;
+using Radzen.Blazor.Rendering;
+using System;
+using System.Drawing;
+using System.Globalization;
+
+namespace Radzen.Blazor
+{
+    /// <summary>
+    /// Displays the appointments in a month day in <see cref="RadzenScheduler{TItem}" />
+    /// </summary>
+    /// <example>
+    /// <code>
+    /// &lt;RadzenScheduler Data="@appointments"&gt;
+    ///     &lt;RadzenMonthView /&gt;
+    /// &lt;/RadzenScheduler&gt;
+    /// </code>
+    /// </example>
+    public partial class RadzenYearView : SchedulerViewBase
+    {
+        /// <inheritdoc />
+        public override string Icon => "view_list";
+
+        /// <inheritdoc />
+        public override string Title
+        {
+            get => Scheduler.CurrentDate.ToString("yyyy", Scheduler.Culture);
+        }
+
+        /// <inheritdoc />
+        [Parameter]
+        public override string Text { get; set; } = "Year";
+
+        /// <summary>
+        /// Specifies the maximum appointnments to render in a slot.
+        /// </summary>
+        /// <value>The maximum appointments in slot.</value>
+        [Parameter]
+        public int? MaxAppointmentsInSlot { get; set; }
```

Comment:
I think it is better the MaxAppointmentsInSlot to have a set default numeric value (as in MonthView) for consistency.

Reply:
This is set in RadzenMonthView.razor and RadzenYearView.razor. Calculated based on the height of the slot.

`@using Radzen.Blazor
@using Radzen.Blazor.Rendering

@inherits SchedulerViewBase

@code {    
    public override RenderFragment Render()
    {
        var appointments = Scheduler.GetAppointmentsInRange(StartDate, EndDate);

        var maxAppointmentsInSlot = 0;

        if (MaxAppointmentsInSlot != null)
        {
            maxAppointmentsInSlot = MaxAppointmentsInSlot.Value;
        }
        else
        {
            var slotHeight = (Scheduler.Height - 60) / 6;
            maxAppointmentsInSlot = Convert.ToInt32(Math.Floor(slotHeight / 24)) - 1;
        }

        return @<CascadingValue Value=@Scheduler>
            <MonthView StartDate=@StartDate EndDate=@EndDate MaxAppointmentsInSlot=@maxAppointmentsInSlot MoreText=@MoreText Appointments=@appointments />
        </CascadingValue>;
    }
}`